### DISCUSSION TITLE
Leistungsanzeige der WR stimmt nicht

### DISCUSSION THREAD

**Discussion by flurfunk0868** [hoylabs/OpenDTU-OnBattery#863] [Q&A]
Hallo zusammen,
habe heute folgendes Problem festgestellt:
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/146464420/caf32b53-ca66-43cc-a194-d1e13cce3db8)

Wie man sieht, bekommt der WR (Tsun 800 W) auf beiden Eingängen Input von meiner Batterie. Laut der obigen Anzeige, kommt aber nur die Hälfte auf der AC-Seite raus (Phase 1). Zuerst hatte ich einen Defekt vermutet. Aber als ich zur Kontrolle die MyStrom-Steckdose ausgelesen habe (Dient mir als redundanter Leistungszähler für meine WR), sah ich dass AC-seitig doch mehr eingespeist wurde. Das bedeutet, die Anzeige im OpenDTU Interface stimmt nicht oder der WR gibt falsche Messwerte aus? Hatte das jemand schonmal?

Folgendes habe ich schon versucht:
- Neustart
- Einschalten und Ausschalten
- Überprüfung der Kabel
- Einstellungen "Einzelne Paneele anzeigen" und "Tagesertrag nullen" an und ausgeschaltet
- Regelung und setzen des Limits über MQTT klappt ohne Probleme

Wie gesagt, in der Realität passt alles aber die angezeigten Messwerte stimmen aus irgendwelchen Gründen nicht... Ist auch meiner Meinung nach ein "neues" Problem

**Comment by spcqike** [comment]
Hallo

ja das Problem ist hinreichend bekannt. Der wechselrichter meldet eine schlechte Effizienz und berechnet damit ne schlechte Leistung. 

Das Problem ist deine 24V Batterie. 

Einfach mal hier suchen. Wurde schon öfter beschrieben. Abhilfe schafft dauerhaft nur ne 48V Batterie

wenn du bei 24V bleiben willst, stell nicht mehr als 200W pro MPPT ein. Also maximal 400W als Limit. 

Grüße

**Reply by flurfunk0868** [reply]
Ok danke für die Info, habe eigentlich nichts vergleichbares unter Diskussionen gefunden...
Komisch finde ich allerdings, dass mein zweiter TSUN an derselben Batterie hängt und auch über dieselbe DTU läuft und diese Probleme nicht hat....

**Reply by spcqike** [reply]
eine generelle Diskussion zur 24V Batterie mit Erklärungen findest du hier
https://github.com/helgeerbe/OpenDTU-OnBattery/issues/613

das Problem mit der Effizienz findest du hier und da in diversen Beiträgen, bspw hier
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/441#discussioncomment-7016161
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/803#discussioncomment-8942796

> Komisch finde ich allerdings, dass mein zweiter TSUN an derselben Batterie hängt und auch über dieselbe DTU läuft und diese Probleme nicht hat....

fährst du den identisch hoch? also mit der gleichen Last? Das Problem taucht wohl erst im Laufe der Zeit auf, wenn die Spannung / Strom ungünstig waren. Dann verhaspelt sich der Hoymiles irgendwie und misst falsch. So genau zu 100% kann es glaube niemand nachvollziehen und erklären, wann es passiert. Ein Neustart des Wechselrichters sollte aber Abhilfe schaffen.

Wie gesagt, die Wechselrichter sind vielleicht für 800W gedacht, 400W pro Eingang, aber eben nur bis zu 12.5A. bei 24V ist da halt bei ~300W schluss. In der Regel bereits früher, da die Spannung niedriger sein kann und so oder so noch einbricht mit steigendem Entladestrom. Dazu kommt, dass die DC(-) Leitungen intern gebrückt sind und der Strom, von DC(1+) auch über DC(2-) zurück fließen kann, der Wechselrichter misst aber auf der (-) Seite. 

Der Betrieb mit Batterien geht, ist vom Hersteller aber nicht vorgesehen. Der Betrieb mit 48V Batterien ist einfacher und stabiler als mit 24V. dazu sind > 30V zwingend nötig, damit die maximale Leistung abgerufen werden kann.

**Reply by flurfunk0868** [reply]
Ok, die Beiträge sind je nachdem was man eintippt schwer zu finden. Unter "Bugs" hab ich gar nicht erst geschaut...

Beide WR werden über ein Node-Red script gesteuert. Hier teile ich einfach den Leistungswert (Aktueller Verbrauch) durch zwei, bevor er als ein Wert an die Wechselrichter geschickt werden. So hat das Ganze auch bis jetzt super funktioniert. Die Leistung wird sauber aufgeteilt und die Regelung funktioniert auch. Meine Erfahrung ist, dass jeder WR maximal 480-500W liefern kann und dabei der Strom bei ca. 10.8-11.05 A (DC-Seitig) limitiert ist. Also knappe 1000 W sind unter idealen Bedingungen zu bekommen. Auf 48 V werde ich in naher Zukunft nicht wechseln, dafür müsste ich dann auch meinen Laderegler und einige andere Komponenten tauschen.
